cartographer/mapping/internal/optimization/optimization_problem_2d.h

```diff
@@ -128,7 +128,8 @@ class OptimizationProblem2D
   MapById<NodeId, NodeSpec2D> node_data_;
   MapById<SubmapId, SubmapSpec2D> submap_data_;
   std::map<std::string, transform::Rigid3d> landmark_data_;
-  sensor::MapByTime<sensor::ImuData> imu_data_;
+  // IMU data is not used in 2D optimization currently and will be empty.
```

Comment:
Please remove. This is the whole point of naming it `empty_imu_data_`. See https://google.github.io/styleguide/cppguide.html#Implementation_Comment_Donts